readme.md

```diff
@@ -529,21 +529,19 @@ Update `remarkConfig`:
   "remarkConfig": {
     "settings": {
       "emphasis": "*",
+      "listItemIndent": "one",
       "strong": "*"
     },
     "plugins": [
       "remark-preset-lint-consistent",
-      "remark-preset-lint-recommended",
-      ["remark-lint-list-item-indent", "space"]
-      ["remark-lint-emphasis-marker", "*"],
-      ["remark-lint-strong-marker", "*"]
+      "remark-preset-lint-recommended"
     ]
   },
   /* … */
 ```
 
-This now includes `settings`, which configures
-[`remark-stringify`][remark-stringify], and explicitly prefers asterisks
+`settings` configures both
+[`remark-stringify`][remark-stringify] and the corresponding rules, and explicitly prefers asterisks
```

Comment:
This is explained better than the previous ones IMO, but I still see room for improvement. E.g., only asterisks is mentioned but not `one`?

Reply:
- [x] I've removed `listItemIndent` from this example: It isn't mentioned in the current prose, either.